composer.json

```diff
@@ -17,6 +17,11 @@
         "wp-coding-standards/wpcs": "2.2.0",
         "phpunit/phpunit": "7.5.2"
     },
+    "config": {
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
```

Comment:
In response to a [failed job](https://app.circleci.com/pipelines/github/studiopress/genesis-custom-blocks/1407/workflows/4e759117-ecd2-4b6c-8988-57feb49a1628/jobs/11059)
)